src/bin/upload-perf.rs

```diff
@@ -18,33 +50,89 @@ fn main() {
         Ok(file) => file,
     };
 
+    let upload_flag = &args[2];
+    let branch = &args[3];
+
     let git_output = Command::new("git")
         .args(&["rev-parse", "HEAD"])
         .output()
         .expect("failed to execute git rev-parse");
     let git_commit_hash = String::from_utf8_lossy(&git_output.stdout);
     let trimmed_hash = git_commit_hash.trim().to_string();
-    println!("uploading hash: {}", trimmed_hash);
+
+    let host =
```

Comment:
Let's move this logic and a generic query function (that `get_last_metrics()` invokes) into src/metrics.rs.  This way we can easily re-use the same default database/password in that file.

Also please change your InfluxDB password, the world now knows it ;)